core/sdk-core/src/main/java/software/amazon/awssdk/core/internal/http/pipeline/stages/MakeHttpRequestStage.java

```diff
@@ -53,8 +63,20 @@ public MakeHttpRequestStage(HttpClientDependencies dependencies) {
 
     private SdkHttpFullResponse executeHttpRequest(SdkHttpFullRequest request, RequestExecutionContext context) throws Exception {
         final AbortableCallable<SdkHttpFullResponse> requestCallable = sdkHttpClient
-                .prepareRequest(request, SdkRequestContext.builder().build());
+            .prepareRequest(request, SdkRequestContext.builder().build());
+
+        long timeoutInMillis = resolveTimeoutInMillis(context.requestConfig()::apiCallAttemptTimeout, apiCallAttemptTimeout);
+
+        TimeoutTracker timeoutTracker = timeSyncTaskIfNeeded(timeoutExecutor, timeoutInMillis, false);
+        timeoutTracker.abortable(requestCallable);
+
+        context.apiCallAttemptTimeoutTracker(timeoutTracker);
+        context.apiCallTimeoutTracker().abortable(requestCallable);
```

Comment:
Is this redundant with line 71?

Reply:
This is intended. line 71 is setting the abortable for `ApiCallAttemptTimeoutTracker` whereas line 74 is setting the abortable for `ApiCallTimeoutTracker`

Reply:
Ah okay I see it